CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Change Log for rocm-cmake
 
+## [0.7.0]
+### Added
+- Client packaging functionality:
+    - `rocm_setup_client_component` sets up a client component, with the given name and values for the component and associated package.
+    - `rocm_install_client_with_symlink` installs a single client target into a specified location, and installs a symlink to that target in a second specified location. 
```

Comment:
`rocm_install_targets` should be used to install targets so they are installed in the correct location. Whats the purpose of the symlinks?

Reply:
`rocm_install_targets` and `rocm_install_symlink_subdir` are now correctly used rather than `rocm_install_client_with_symlink`.